tools/mock-okta/util.js

```diff
@@ -203,10 +204,14 @@ util.mapRequestToCache = (req) => {
     data.responseMode = query.response_mode;
     data.state = query.state;
     data.nonce = query.nonce;
-    req.url = req.url
-      .replace(`state=${query.state}`, 'state=STATE')
-      .replace(`nonce=${query.nonce}`, 'nonce=NONCE');
 
+    // If playing back the tapes, replace the state in req.url with a known state
+    if (!record) {
```

Comment:
nit: Can we add an `@arg` doc param above to describe what this is?

Reply:
Done